Jenkinsfile.conda

```diff
@@ -1,2 +1,2 @@
 @Library('JenkinsShared')_
-CondaPipeline(["ts_config_ocs"], "ts-ess", "lsst.ts.ess", "noarch")
+CondaPipeline(["ts_config_ocs"], "ts-ess-csc", "lsst.ts.ess.csc", "noarch")
```

Comment:
Please revert this change. The cycle build expects the conda package to be called `ts-ess`. I created DM-32978 for doing this since it requires more changes than just this file.

Reply:
Done. Thanks for filing a ticket for this!